addons/l10n_eu_service/models/eu_tax_map.py

```diff
@@ -0,0 +1,2193 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+EU_TAX_MAP = {
+    'AT10.0BE':6.0,
```

Comment:
rather use tuple as keys that concatenating:
('AT', 10.0, 'BE'): 6.0

A small comment on top of the dict telling what each value is would also be nice, for clarity.

Reply:
👍